QuickStart.md

```diff
@@ -0,0 +1,219 @@
+
+Now that we have an object, let's defined the controller which actually sends the object to the server.  Open the file MyWebServiceController.java, and Add Code until it looks like the following:
+```
+com.mycompany.mywebservice.core
+
+import org.apache.http.HttpHeaders;
+import com.adobe.ride.core.controllers.RestApiController;
+import com.adobe.ride.core.types.MimeTypes;
+import com.adobe.ride.utilities.model.ModelObject;
+
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.filter.Filter;
+import io.restassured.http.Method;
+import io.restassured.response.Response;
+import io.restassured.specification.ResponseSpecification;
+
+public class MyWebServiceController extends RestApiController {
+
+  protected static RequestSpecBuilder getDefaultReqSpecBuilder(Filter... filters) {
+    RequestSpecBuilder builder;
+    builder = RestApiController.getRequestBuilder(true, filters);
+    builder.addHeader(HttpHeaders.ACCEPT, MimeTypes.APP_JSON.toString());
+    builder.addHeader(HttpHeaders.CONTENT_TYPE, MimeTypes.APP_JSON.toString());
+    return builder;
+  }
+
+  private static Response firePOSTCall(String objectPath, ModelObject object,
+      ResponseSpecification expectedResponse) {
+    RequestSpecBuilder reqBuilder = getDefaultReqSpecBuilder();
+    reqBuilder.setBody(object.getObjectMetadata());
+
+    Response response = fireRestCall("SampleService", objectPath, reqBuilder,
+          expectedResponse, method.POST);
+
+    return response;
+  }
+}
+```
+
+Ok, now we have our config information setup and our basic object and controller code.  Let's compile it into a library.
+
+From either your IDE or the location of your pom in terminal/command line window, run the standard maven install command:
+``` mvn clean install ```
+
+## Test Project and Code
+
+Now we have a distributable library that you can post on your internal artifact repository (once you get it into a shape you like, for now we will use your locally compiled version).
+
+Let's create a project to write our tests.  In your IDE create a new Java Project with a groupId and artifactId different from your library, something like com.mycompany.mywebservice.automation /  webservice-tests
+
+In your pom, add a dependency for the library you just created:
+
+```
+	<dependency>
+      <groupId>com.mycompany.mywebservice</groupId>
+      <artifactId>webservice-automation-library</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
+    </dependency>
+```
+
+Add the following package/class in the test source of your project
+
+src/test/java
+	- com.mycompany.mywebservice.tests
+		- Basic\_Test\_IT.java
+		
+Open Basic\_Test\_IT.java and add code until it looks like the following:
```

Comment:
add code until it looks like the following -> add the following code

Reply:
I believe this comment is no longer applicable because I switched to html formatting here.  Can you take another look?